script/crossbinary-default

```diff
@@ -33,12 +34,12 @@ for OS in ${OS_PLATFORM_ARG[@]}; do
 done
 
 
-# Build arm64 binaries
-OS_PLATFORM_ARG=(linux)
-OS_ARCH_ARG=(arm64)
-for OS in ${OS_PLATFORM_ARG[@]}; do
-  for ARCH in ${OS_ARCH_ARG[@]}; do
-    echo "Building binary for $OS/$ARCH..."
-    GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 $GO_BUILD_CMD "$GO_BUILD_OPT" -o "dist/traefik_$OS-$ARCH" ./cmd/traefik/
-  done
-done
+## Build arm64 binaries
+#OS_PLATFORM_ARG=(linux)
+#OS_ARCH_ARG=(arm64)
+#for OS in ${OS_PLATFORM_ARG[@]}; do
+#  for ARCH in ${OS_ARCH_ARG[@]}; do
+#    echo "Building binary for $OS/$ARCH..."
+#    GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 $GO_BUILD_CMD "$GO_BUILD_OPT" -o "dist/traefik_$OS-$ARCH" ./cmd/traefik/
+#  done
+#done
```

Comment:
Oops, my bad... accidentally committed something that should not have been...  will revert ASAP